cylc/rose/__init__.py

```diff
@@ -14,44 +14,44 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
-Cylc Rose
-=========
-
-Cylc Rose is the bridge between Rose suite configurations and Cylc workflows.
-
-Cylc Rose allows ``cylc install`` to replace the ``rose suite-run``
-command (present in Rose versions 2019.01 and earlier). It reads the
-``rose-suite.conf`` file and:
+Rose Suite Configurations
+=========================
 
-- Makes environment and template variables available
-  to Cylc.
-- Installs files.
-- Records information in about the configuration installed in
-  ``~/cylc-run/<workflow>/opt/rose-suite-cylc-install.conf``
+This plugin is activated by the existence of a Rose Suite Configuration file
+(called ``rose-suite.conf``) in the workflow definition directory which allows
+you to:
 
+- Set template variables used in the workflow definition.
+- Configure files to be installed via ``cylc install``.
+- Define environment variables for the Cylc scheduler.
 
 The following Cylc commands will read Rose Suite Configurations:
 
+- ``cylc install``
 - ``cylc validate``
 - ``cylc graph``
 - ``cylc list``
 - ``cylc config``
 
+.. Note::
+
+   Cylc Rose allows ``cylc install`` to replace the ``rose suite-run``
+   command (present in Rose versions 2019.01 and earlier).
+
 
 Rose Config
```

Comment:
This doesn't make much sense now. I suggest changing this to "Configuration" or ""Configuration file contents"